gossip/c_event_callbacks.go

```diff
@@ -80,18 +87,26 @@ func (s *Service) processSavedEvent(e *inter.EventPayload, es *iblockproc.EpochS
 		return errWrongMedianTime
 	}
 
+	begin := ctx.Value("startOfValidation").(time.Time)
+	blockValidationTimer.Update(time.Since(begin))
+
 	// aBFT processing
-	return s.engine.Process(e)
+	err = s.engine.Process(e)
```

Comment:
Can you explain why we need to do this refactoring?

Reply:
We don't need it )